### ISSUE TITLE
Enter one-line summary

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#248]
```
This issue was created by revision 823a33e01ad1.

Manage the DOB and State in user models
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 19 Dec 2012 at 2:59

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision d1f57372b9b0.

State, photo coded
```

Original comment by `eric.mae...@gmail.com` on 19 Dec 2012 at 3:29

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision f4297282129a.
```

Original comment by `eric.mae...@gmail.com` on 19 Dec 2012 at 4:49
- Changed state: **Fixed**
